docs/content/concepts/io-management/io-managers.mdx

```diff
@@ -66,6 +66,16 @@ The default IO manager, <PyObject module="dagster" object="fs_io_manager" />, st
 
 Dagster provides out-of-the-box IO managers for popular storage systems: AWS S3 (<PyObject module="dagster_aws.s3" object="s3_pickle_io_manager" />), Azure Blob Storage (<PyObject module="dagster_azure.adls2" object="adls2_pickle_io_manager" />), GCS (<PyObject module="dagster_gcp.gcs" object="gcs_pickle_io_manager" />), and Snowflake (<PyObject module="dagster_snowflake" object="build_snowflake_io_manager" />) - or you can write your own.
 
+Dagster provides a feature-rich base class for filesystem-based IO Managers: <PyObject module="dagster" object="UPathIOManager" />. It's compatible with both local and remote filesystems (like S3 or GCS) by using `universal-pathlib` and `fsspec`. The full list of supported filesystems can be found [here](https://github.com/fsspec/universal_pathlib#currently-supported-filesystems-and-schemes). The `UPathIOManager` also has other important features:
```

Comment:
This is an introductory section, so we've been careful in the past to not get into too much detail - readers who want to understand what it supports can find the details inside the API doc for UPathIOManager.

I would recommend something like:

> Dagster provides out-of-the-box IO managers for popular storage systems: AWS S3... You can also write your own: either from scratch or by extending the `UPathIOManager` if you want to store data in an `fsspec`-supported filesystem.

Reply:
done